src/docs/organization.yaml

```diff
@@ -218,7 +218,49 @@ paths:
                                     message:
                                         type: string
                                         example: 'Invalid Token: No authorization token was found'
-
+    /api/organization/{id}/sub:
+        post:
+            tags:
+                - Organization
+            summary: Delete organization sub
+            description: Delete organization sub
+            operationId: deleteSubscription
+            consumes:
+                - application/json
+            produces:
+                - application/json
+            parameters:
+                - in: path
+                  name: id
+                  description: The id of the organization
+                  required: true
+                  type: string
+            responses:
+            # i have 2 (404) code statue in my code. how should i handle it here ? 
```

Comment:
There is no need to add 2 (404) responses. I see that your response objects also have the same structure `({error: "..."})`, and since you are providing an example, I think you don't have to add anything extra.